bayesian/api/api_v2.py

```diff
@@ -199,6 +197,9 @@ def post():
         response_template: Tuple = namedtuple("response_template", ["message", "status", "headers"])
         input_json: Dict = request.get_json()
         ecosystem: str = input_json.get('ecosystem')
+        user_agent = request.headers.get('User-Agent', None)
+        manifest_hash = str(request.headers.get('manifest_hash', None))
```

Comment:
Why are we overloading request header? shouldn't this be optional filed of request payload?